django_apscheduler/admin.py

```diff
@@ -39,6 +46,20 @@ def average_duration(self, obj):
 
     average_duration.short_description = "Average Duration (sec)"
 
+    actions = ['run_selected_jobs']
+
+    def run_selected_jobs(self, request, queryset):
+        scheduler = BackgroundScheduler()
+        scheduler.add_jobstore(DjangoJobStore(), "default")
+        scheduler.start()
+        for item in queryset:
+            job = scheduler.get_job(item.id, "default")
+            if job:
+                scheduler.modify_job(job.id, "default", next_run_time=timezone.now())
```

Comment:
It *might* be possible to achieve the same result by just selecting the relevant job entry in `DjangoJob` using a Django query and updating the `next_run_time` manually, though I have not tested this.

Reply:
I tried, it doesn't work, changing next_run_time will not wakeup existing scheduler, so job will not be run, it will try to run at next wakeup but if misfire_grace_time is not set job will marked as missed. 
another idea add fake every minute job, and set misfire_grace_time=61 to real job, but I don't like this solution. 
My idea was create one more scheduler, start it, run tasks, stop scheduler. 
If you will create "date" job scheduler will delete it from storage,  so instance of DjangoJob will be deleted, so DjangoJobExecution will not be created.
BTW I have one more idea: add soft delete to DjangoJob model, in this case when scheduler will delete DjangoJob instance, we will mark row is_deleted=True, and update get_jobs to return jobs with is_deleted=False, what do you think?

Reply:
> BTW I have one more idea: add soft delete to DjangoJob model, in this case when scheduler will delete DjangoJob instance, we will mark row is_deleted=True, and update get_jobs to return jobs with is_deleted=False, what do you think?

I am a little hesitant to change the semantics of how APScheduler's job stores work, but I can appreciate the value in keeping historical job executions - perhaps create a separate GitHub issue to track and discuss this change.